p5js/SoundClassification/SoundClassification_speechcommand/sketch.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2018 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ml5 Example
+Sound classification using speech-commands and p5.js
+This example uses a callback pattern to create the classifier
+=== */
+
+// Initialize a sound classifier method with speech-commands model. A callback needs to be passed.
+let classifier;
+// Options for the speech-commands model, the default probabilityThreshold is 0
+const options = { probabilityThreshold: 0.7 };
+// Two variable to hold the label and confidence of the result
+let label;
+let confidence;
+
+function preload() {
+  // Load speech-commands sound classifier model
+  classifier = ml5.soundClassifier('speech-commands', options);
```

Comment:
This is a nitpicky point, but we have used dashes in model names yet, it's mostly been CamelCase (but not actually case sensitive). i.e. `'SpeechCommands`? Where did this model come from, is there a specific name like "MobileNet"? Is that "Speech Commands"?